### PR TITLE
Live update 4/2023

### PR DESCRIPTION
**Pull request by kpodemski**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

File: content/news/public-oss-demo/2023/upcoming-live-update-4-2023.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+subtitle: "Let's meet for the fourth Live Update in 2023"
+date:   2023-04-11
+authors: [ kpodemski ]
+icon: icon-video-camera
+tags: [contribute, event, community, live-update]
+---
+
+The fourth Live Update for the PrestaShop Project will happen on **Wednesday, April 12th, 2023** at [4 pm (Paris time)](https://time.is/1600_12_Apr_2023_in_Paris) on [YouTube](https://www.youtube.com/watch?v=SVbAQ__A6ns)!
+
+Tomorrow we'll discuss what's going on in the project, show you recent changes in the developer documentation, and, most importantly, [@eternoendless](https://github.com/eternoendless) will share exciting news about a project organization.
```
Comment (eternoendless) [inline content/news/public-oss-demo/2023/upcoming-live-update-4-2023.md]:
```suggestion
Tomorrow we'll discuss what's going on in the project, show you recent changes in the developer documentation, and, most importantly, [@eternoendless](https://github.com/eternoendless) will share exciting news about the project's organization.
```

Comment (coffrep) [inline content/news/public-oss-demo/2023/upcoming-live-update-4-2023.md]:
```suggestion
Tomorrow we'll discuss what's going on in the project, show you recent changes in the developer documentation, and, most importantly, [@eternoendless](https://github.com/eternoendless) will share exciting news about the project's organization.
```

File: content/news/public-oss-demo/2023/upcoming-live-update-4-2023.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Remember that at the end of every meeting, there is a Q&A session. You can ask your questions beforehand.
```
Comment (eternoendless) [inline content/news/public-oss-demo/2023/upcoming-live-update-4-2023.md]:
```suggestion
Remember that at the end of every meeting, there is a Q&A session. You can ask your questions during the presentation or beforehand.
```

File: content/news/public-oss-demo/2023/upcoming-live-update-4-2023.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+**If you have some, about things related to the open source project**, feel free to ask [using this form](https://forms.gle/FWazuZnXBtFPauFZ7). They will be answered during the Q&A session.
```
Comment (eternoendless) [inline content/news/public-oss-demo/2023/upcoming-live-update-4-2023.md]:
```suggestion
**If you have questions related to the open source project**, feel free to let us know in advance [using this form](https://forms.gle/FWazuZnXBtFPauFZ7). We will do our best to answer them during the Q&A session.
```

**Review by coffrep — APPROVED** [review]
Hi @kpodemski - approving with one very minor change.

**Review by eternoendless — COMMENTED** [review]
Some minor suggesstions, but otherwise ok!

**Pull request merged**